MeetupPlatformApi/Entities/MeetupEntity.cs

```diff
@@ -1,14 +1,22 @@
-﻿namespace MeetupPlatformApi.Entities;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.Entities;
 
 public class MeetupEntity
 {
+    [Required]
```

Comment:
Fields with not-nullable .Net types are required by default, so there is no need to specify `[Required]` for `Id`, `Name` and `Description` fields. But, if You guys insist, we can keep them and make it our convention